test/python/providers/test_backendproperties.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Base TestCase for testing Providers."""
+
+import unittest
+import datetime
+from qiskit.test.mock import FakeOpenPulse2Q
+from qiskit.test.mock import FakeProvider
+from qiskit.test import QiskitTestCase
+
+from qiskit.providers.models.backendproperties import Nduv, Gate, BackendProperties
+
+class BackendpropertiesTestCase(QiskitTestCase):
```

Comment:
Please test some of the failure modes. Check that bad input raises `PulseError`s